SparkPost/2web2ui
# SA-1773 - (UI) Release Advanced Comparators

[SA-1773](https://sparkpost.atlassian.net/browse/SA-1773)

### What Changed
- Removes references to the `"allow_report_filters_v2"` account UI option
- Removes the old state reducer for handling filters
- Removes the old UI for handling filters
- Updates Cypress tests, removing tests for old filter UI and removing references to fixture files with feature flags
- Centralized use of the `dehydrateFilters` helper function

### How To Test
- Verify adding/removing filters works as expected without any particular feature flag for report builder
- Verify filters can be removed using the tags rendered on the page
- Verify old filters applied via the `filters=` query param are still re-mapped to `query_filters=` 

### To Do
